x/concentrated-liquidity/msg_server_test.go

```diff
@@ -57,7 +57,7 @@ func (suite *KeeperTestSuite) TestCreateConcentratedPool_Events() {
 
 	for name, tc := range testcases {
 		suite.Run(name, func() {
-			suite.Setup()
+			suite.SetupTest()
```

Comment:
What is the purpose of Setup? I only ever use SetupTest as well since thats what everything else uses, but I feel like we should just get rid of it if it isn't useful right?

Reply:
Setup runs at the app testing level. This is what's done for all modules.

SetupTest implements custom logic for each test suite

I put the parameter update into Setup because it was applicable to most modules